src/en/clean-copy/02-Part I. From Sumerians to Sadi Carnot/11.md

```diff
@@ -2,36 +2,36 @@
 ##### Historical period: the 18th century CE
 ##### Scene of Action: Burton upon Trent
 
-In the beginning of the 18th century, London was the world's brewing capital by a vast margin. Hansa hadn't recovered after the Thirty Years' War of 1618-1648; the Netherlands, which was severed by the wars of religion even stronger than the League, at the end of the 17th century had rebuilt its economics, but not the brewing industry — because of multiple reasons, from high taxes to the grape wine and strong alcohol expansion throughout Northern Europe. The growing popularity of porter strengthened London's superiority even further.
+In the beginning of the 18th century, London was the world's brewing capital by a vast margin. Hansa hadn't recovered after the Thirty Years' War of 1618-1648 and the Netherlands was severed by the wars of religion even more so than the League. By the  end of the 17th century it had rebuilt its economics, but not the brewing industry for a host of reasons: high taxes compared to grape wine and strong alcohol expansion throughout Northern Europe. The growing popularity of porter strengthened London's superiority even further. 
```

Comment:
I don't actually know whether beer taxes were higher than those on wine. They were just high. Changed it to 'high taxes and strong alcohol expansion throughout Northern Europe, to name a few' — please review